### ISSUE TITLE
Undo triggers in wrong file when using split windows

### ISSUE THREAD

**Issue by bbonf** [cursorless-dev/cursorless#2063]
Not sure if this is a bug or a configuration issue:

When using a split editor, issuing an undo command will always undo the last edit that was done in the current file.
But when working with two files in a split view, this is often not the desired outcome. I would expect it to always undo the last command/edit across all files. This is especially annoying when running into a recognition error, or when using the wrong hat.
In my setup it's trivial to reproduce, but please let me know if there's more info I could provide.

That being said, many thanks for your work on cursorless, it's really great! 👏

**Comment by bbonf** [comment]
It's a tricky point, because I can't imagine most keyboard users would be happy with undo-s that switch around between the visible split editors. When using the keyboard, I think it does make more sense to undo only in the current editor. 
Anyway, I think this requires some more consideration.

**Comment by pokey** [comment]
> You can also use a Cursorless custom action if you want to be able to undo and redo across a split
> 
> `"undo blue air"`

How did that never occur to me 😅. Are you using that?

Btw if you unmap "undo that", then for actions that only target one split, saying "undo that" would just work in the right split 🤔. We could even support commands that group by editor and run once for each editor and it would just undo in multiple splits if necessary 😄

**Comment by pokey** [comment]
that's great. added myself 👍

**Comment by pokey** [comment]
Just did the same for save:

```csv
disk, workbench.action.files.save
```

**Comment by AndreasArvidsson** [comment]
> Just did the same for save:
> 
> ```
> disk, workbench.action.files.save
> ```
> 
> So now I can say "round wrap air" in opposite split then "disk that"

Nice. I have `disk files` to save all open files which I use kinda similarly.

**Comment by pokey** [comment]
> Nice. I have disk files to save all open files which I use kinda similarly.

Yeah that's what I usually use but I like the precision of this one, because sometimes that triggers annoying save untitled popup

**Comment by AndreasArvidsson** [comment]
Since this is a vscode feature and we already have an issue for phrase level undue I will close this issue.
